### PR TITLE
[plutono] bump plutono and postgresql image versions

### PR DESCRIPTION
**Pull request by thgrs**

### CODE REVIEW

File: system/plutono/values.yaml [hunk 1]
```diff
@@ -29,7 +29,7 @@ plutono:
   tls: true
   disco: true
   image: credativ/plutono
-  image_version: v7.5.26
+  image_version: v7.5.29
```
Comment (richardtief) [inline system/plutono/values.yaml:32]: As discussed, this can be `latest`.

File: system/plutono/Chart.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 description: A Helm chart for SAP's CCloud Plutono 7.5.x deployment
 name: plutono
-version: 0.2.6
+version: 0.2.7
 dependencies:
   - name: postgresql
     repository: file://../../common/postgresql
```
Comment (viennaa) [inline system/plutono/Chart.yaml:7]: unrelated, but why isn't this using `https://charts.eu-de-2.cloud.sap`

Comment (richardtief) [reply system/plutono/Chart.yaml:7]: https://github.com/sapcc/helm-charts/blob/641d765d4e7839a272f545625f824aefa0a2c357/common/postgresql/Chart.yaml#L2

File: system/plutono/values.yaml (hunk 1 above)
Comment (thgrs) [reply system/plutono/values.yaml]: updated

**Review by richardtief — APPROVED** [review]
Thanks!

**Pull request merged**